programs/operator/src/cli/command.rs

```diff
@@ -75,6 +75,11 @@ pub mod genesis {
         /// Supported proof types: groth16, plonk.
         #[clap(long, short = 'p', value_parser = super::parse_proof_type, default_value = "plonk")]
         pub proof_type: super::SupportedZkAlgorithm,
+
+        /// Running with a private cluster or not
+        /// If true, the operator will use the private cluster configuration.
+        #[clap(long, default_value = "false")]
+        pub private_cluster: bool,
```

Comment:
Proof type and private cluster should be refactored to a new struct called something like SP1Args, then only that should be imported to all the other args and be flattened. I'm also open to other refactors that remove this duplication.

Reply:
probably, given that we don't have a timeline on when it will be gone. + It won't be too much effort

Reply:
Fair enough! Added it now